code/modules/shuttle/ripple.dm

```diff
@@ -0,0 +1,22 @@
+/obj/effect/ripple
+	name = "hyperspace ripple"
+	desc = "Something is coming through hyperspace, you can see the \
+		visual disturbances. It's probably best not to be on top of these \
+		when whatever is tunneling comes through."
+	icon = 'icons/turf/floors/ripple.dmi'
+	icon_state = "ripple"
+	anchored = TRUE
+	density = FALSE
+	smooth = SMOOTH_TRUE
+	layer = RIPPLE_LAYER
+	alpha = 0
```

Comment:
if you make this a child of `temp` then you can just have the duration set to `duration = 3 * SHUTTLE_RIPPLE_TIME`